expenses/controllers/UserController.py

```diff
@@ -1,6 +1,29 @@
+from graphene import Enum
 from django.contrib import auth
+from django.contrib.auth.models import Group
+from seaport.models import User
+from expenses.rules import is_exec
 
 
+class UserAccessLevel(Enum):
+    NONE = 0
+    MEMBER = 1
+    EXEC = 2
+    ADMIN = 3
+
+
+def get_access_level(user):
+    if not user or not user.is_active:
+        return UserAccessLevel.NONE
+
+    groups = user.groups
+
+    if groups.filter(name="admin").exists():
+        return UserAccessLevel.ADMIN
+    elif groups.filter(name="exec").exists():
+        return UserAccessLevel.EXEC
+    return UserAccessLevel.MEMBER
```

Comment:
What exactly indicates that a user has the NONE access level?  I see where you catch that above but the default case here being MEMBER seems like it could be a potential problem

Reply:
Essentially, it seems like there is a member group [here](https://github.com/HackGT/piranha/blob/e6d0cda1634a98ae83ef2aa78fb08a381c2d60cb/seaport/models.py#L23):
```python
        member_group = Group.objects.get(name="member")
```

To be safe, I would suggest adding an explicit case to check for the member group here, and make the base case the NONE access level

Reply:
Fixed